src/sys/windows/event.rs

```diff
@@ -14,25 +14,25 @@ pub fn token(event: &Event) -> Token {
 }
 
 pub fn is_readable(event: &Event) -> bool {
-    if is_hup(event) {
+    if is_error(event) {
```

Comment:
Why are you returning `true` if there is an error? Same for `is_writable`.

Reply:
I would expect this to be correct. An error would imply that the socket is readable / writable and will return an error on that operation.